packages/react-email/src/components/shell.tsx

```diff
@@ -22,97 +25,103 @@ export const Shell = ({
   markup,
   activeView,
   setActiveView,
+  theme,
+  setTheme,
 }: ShellProps) => {
   const [sidebarToggled, setSidebarToggled] = React.useState(false);
   const [triggerTransition, setTriggerTransition] = React.useState(false);
 
   return (
-    <div className="flex bg-black text-white flex-col h-screen overflow-x-hidden">
-      <div className="flex lg:hidden items-center px-6 justify-between h-[70px] border-b border-slate-6">
-        <div className="h-[70px] flex items-center">
-          <Logo />
-        </div>
+    <ThemeProvider setTheme={setTheme} theme={theme}>
```

Comment:
Do we really need a Provider for the theme here? Isn't the theme just meant for the preview of the email?

Reply:
Thinking about it, we don't really need a theme provider to begin with, it's just doing the same thing as the search params, so I've removed it so now it is just the search params handling the state.